docs/guides/how_to_guides/configuring_data_contexts/how_to_instantiate_a_data_context_on_an_databricks_spark_cluster.rst

```diff
@@ -0,0 +1,198 @@
+
+   project_config = DataContextConfig(
+       config_version=2,
+       plugins_directory=None,
+       config_variables_file_path=None,
+
+       datasources={
+           "my_spark_datasource": {
+               "data_asset_type": {
+                   "class_name": "SparkDFDataset",
+                   "module_name": "great_expectations.dataset",
+               },
+               "class_name": "SparkDFDatasource",
+               "module_name": "great_expectations.datasource",
+               "batch_kwargs_generators": {},
+           }
+       },
+       stores={
+       "expectations_store": {
+           "class_name": "ExpectationsStore",
+           "store_backend": {
+               "class_name": "TupleFilesystemStoreBackend",
+               "base_directory": "REPLACE ME",  # TODO: replace with your value
+           },
+       },
+       "validations_store": {
+           "class_name": "ValidationsStore",
+           "store_backend": {
+               "class_name": "TupleFilesystemStoreBackend",
+               "base_directory": "REPLACE ME",  # TODO: replace with your value
+           },
+       },
+       "evaluation_parameter_store": {"class_name": "EvaluationParameterStore"},
+    },
+    expectations_store_name="expectations_store",
+    validations_store_name="validations_store",
+    evaluation_parameter_store_name="evaluation_parameter_store",
+    data_docs_sites={
+       "local_site": {
+           "class_name": "SiteBuilder",
+           "store_backend": {
+               "class_name": "TupleFilesystemStoreBackend",
+               "base_directory": "REPLACE ME",  # TODO: replace with your value
+           },
+           "site_index_builder": {
+               "class_name": "DefaultSiteIndexBuilder",
+               "show_cta_footer": True,
+           },
+       }
+    },
+    validation_operators={
+       "action_list_operator": {
+           "class_name": "ActionListValidationOperator",
+           "action_list": [
+               {
+                   "name": "store_validation_result",
+                   "action": {"class_name": "StoreValidationResultAction"},
+               },
+               {
+                   "name": "store_evaluation_params",
+                   "action": {"class_name": "StoreEvaluationParametersAction"},
+               },
+               {
+                   "name": "update_data_docs",
+                   "action": {"class_name": "UpdateDataDocsAction"},
+               },
+           ],
+       }
+    },
+    anonymous_usage_statistics={
+     "enabled": True
+    }
+    )
+
+   context = BaseDataContext(project_config=project_config)
+
+#. **Install Great Expectations on your Databricks Spark cluster.**
+
+   Copy this code snippet into a cell in your Databricks Spark notebook and run it:
+
+   .. code-block:: python
+
+      dbutils.library.installPyPI("great_expectations")
+
+
+#. **Configure an Expectation store in DBFS**
```

Comment:
I'm actually wondering if these sections are necessary, or if you can just put the placeholders in the code snippet, like so:

```

"expectations_store": {
           "class_name": "ExpectationsStore",
           "store_backend": {
               "class_name": "TupleFilesystemStoreBackend",
               "base_directory": "/FileStore/expectations/",  # TODO: replace with the path to your store
           },
       },

```

And then just call out right above the snippet that the paths to expectation stores, validation result stores and data docs need to be replaced with the respective paths.